### ISSUE TITLE
Error upload .......  to Dropbox: Expecting a file upload

### ISSUE THREAD

**Issue by NielsEhv** [michaeldewildt/wordpress-backup-to-dropbox#175]
I can't seem to make normal backups anymore. Tried unlinking/re-linking but all backups fail with the message "The backup is having trouble uploading files to Dropbox, it has failed 10 times and is aborting the backup".

The reason is "Error upload .......  to Dropbox: Expecting a file upload".

I don't understand what the reason is for this error to happen.

09:08:25: A fatal error occured: The backup is having trouble uploading files to Dropbox, it has failed 10 times and is aborting the backup.
09:08:25: Processed 931 files.
09:08:25: Error uploading '/customers/f/0/2/domainname.com/httpd.www/wp-content/plugins/jetpack/images/jetpack-protect-shield.svg' to Dropbox: Expecting a file upload
09:08:23: Error uploading '/customers/f/0/2/domainname.com/httpd.www/wp-content/plugins/jetpack/images/screenshots/manage-sm.png' to Dropbox: Expecting a file upload
09:08:21: Error uploading '/customers/f/0/2/domainname.com/httpd.www/wp-content/plugins/jetpack/images/screenshots/manage.png' to Dropbox: Expecting a file upload

**Comment by niconoe** [comment]
Same issue for me. Have tried removing all plugin config (in the database) and multiple Dropbox accounts. Don't know if it could be hosting-related, this website uses One.com.

**Comment by NielsEhv** [comment]
My website us also hosted by one.com!

**Comment by magic282** [comment]
Same issue here. I use a VPS to host the wordpress.

**Comment by duikb00t** [comment]
I have the same issue. Im just running my site here on localhost. Is there already a fix for this?

**Comment by benjamardo** [comment]
Same issue and also on one.com.

**Comment by NiniLaNouille** [comment]
It's not an issue due to one.com but from your php version (which is linked to your host)
I was with ovh with the same issue...
You just have to change your php version from a 5.6 to a 5.5 (see with one.com how to do it) and it will work fine.

**Comment by pimhooghiemstra** [comment]
@NiniLaNouille, you are right! I am glad I found this thread. Just checked my hosting package for the site I was having trouble with and it turned out it was running PHP 5.6. Just switched it back to version 5.5 and the backup seems to start working! Great!

**Comment by NielsEhv** [comment]
Hey!

 

Have you already seen it? http://packshot-web.fr/reason.php?olvh

 

niels@eijndhoven.eu

**Comment by NielsEhv** [comment]
Hello!

 

Important message, visit http://ruderclub-forggensee.de/state.php?3gx7

 

niels@eijndhoven.eu
